
Running tests...
----------------------------------------------------------------------
E
======================================================================
ERROR [0.010s]: OBD_tests (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: OBD_tests
Traceback (most recent call last):
  File "/usr/lib/python3.6/unittest/loader.py", line 153, in loadTestsFromName
    module = __import__(module_name)
  File "/home/not-yeshwanth-reddy/Documents/Sem6/Capstone/OBD/src/unittest/python/OBD_tests.py", line 4, in <module>
    import OBD
  File "/home/not-yeshwanth-reddy/Documents/Sem6/Capstone/OBD/src/main/python/OBD.py", line 25, in <module>
    import cv2
ModuleNotFoundError: No module named 'cv2'


----------------------------------------------------------------------
Ran 1 test in 0.010s

FAILED (errors=1)

Generating XML reports...
